### PR TITLE
Add more topic ideas to README

### PR DESCRIPTION
**Pull request by hilton**
> I will keep my eyes open on what people work on and try to ask them to write about specific topics. But so far, I had problems identifying such topics.

Here are a few more ideas!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,15 @@ In practice, this is likely to be one of:
 * Fun tech ideas you talked about over beers
 * A demo of something cool, however pointless
 
+More topic ideas:
+
+* Annoucement for a forthcoming Signavio Tech Talk meet-up
+* Presentation slides or photos of the previous Signavio Tech Talk meet-up
+* Book review
+* Masters thesis summary
+* Review of a Berlin tech meetup event
```
Comment (TimKam) [inline README.md]: Or _any_ tech meet-up event?

File: README.md [hunk 2]
```diff
@@ -16,6 +16,15 @@ In practice, this is likely to be one of:
 * Fun tech ideas you talked about over beers
 * A demo of something cool, however pointless
 
+More topic ideas:
+
+* Annoucement for a forthcoming Signavio Tech Talk meet-up
```
Comment (TimKam) [inline README.md]: Type: *Announcement*
Also, I think we should either cover all (tech) meet-ups or none.
Should we have a vote on this on Slack?

**Comment by hilton** [comment]
@TimKam Changes made.

File: README.md [hunk 3]
```diff
@@ -16,6 +16,15 @@ In practice, this is likely to be one of:
...
+* Presentation slides or photos of the previous Signavio Tech Talk meet-up
+* Book review
+* Masters thesis summary
```
Comment (katsharp) [inline README.md]: I think we should encourage this more, also for bachelors

**Pull request merged**